pyqg/kernel.pyx

```diff
@@ -145,14 +156,23 @@ cdef class PseudoSpectralKernel:
         difftout = self._empty_real()
         self._dummy_ifft_out = difftout
 
+        # time stuff
+        self.dt = 0.0
```

Comment:
My approach here was simply to initialize all the variables in the kernel. They can (and should) be overwritten later.